src/ansys/fluent/core/launcher/error_handler.py

```diff
@@ -101,12 +102,14 @@ def _process_invalid_args(dry_run, fluent_launch_mode, argvals):
     argvals: dict
         Local arguments.
     """
-    if dry_run and fluent_launch_mode != LaunchMode.CONTAINER:
+    import ansys.fluent.core.launcher.pyfluent_enums as enums
+
+    if dry_run and fluent_launch_mode != enums.LaunchMode.CONTAINER:
         launcher_utils.logger.warning(
             "'dry_run' argument for 'launch_fluent' currently is only "
             "supported when starting containers."
```

Comment:
The objective is to make code like this is redundant since each launcher type specifies its construction interface. We shall not have such a generalised argument validation function.

Reply:
Yes we will address this issue in next PR. Thank you.